### ISSUE TITLE
Make sure all icons which represent state are accessible

### ISSUE THREAD

**Issue by acelaya** [shlinkio/shlink-web-component#76]
Some icons in the app are used to provide important information. These icons should be announced to screen readers.

- DomainStatusIcon
